.github/workflows/fpga.yml

```diff
@@ -106,9 +110,21 @@ jobs:
           path: /tmp/caliptra-fpga-kmod/
           retention-days: 1
 
+  # this workflow uploads a file named caliptra-unified-<rom-ref>-<fw-ref>-<hw-version>-<artifact-suffix>
+  versioned-checkout-build:
+    uses: ./.github/workflows/versioned-checkout.yml
+    with:
+      rom-ref: "${{ inputs.rom-ref }}"
+      rom-logging: "${{ inputs.rom-logging }}"
+      fw-ref: "${{ inputs.fw-ref }}"
+      hw-version: "${{ inputs.hw-version }}"
+      workflow_call: "${{ inputs.workflow_call }}"
+      artifact-suffix: "${{ inputs.artifact-suffix }}"
```

Comment:
Is it possible to isolate the FPGA jobs to the nightly run? 

I think running all of these tests on each PR will effectively DDOS the FPGA infra.